### PR TITLE
Add openshift/tektoncd-pipeline v0.16.3 jobs

### PR DESCRIPTION
**Pull request by nikhil-thomas**
Add jobs for ocp-4.4, ocp-4.5 and ocp-4.6

Add image mirroring for tektocd-pipeline v0.16

Signed-off-by: Nikhil Thomas <nikthoma@redhat.com>

### CODE REVIEW

**Comment by chmouel** [comment]
/lgtm
/approve

Thanks 👍🏼🔥

File: core-services/image-mirroring/tekton/mapping_tekton_v0_16_quay [hunk 1]
```diff
@@ -1,10 +1,10 @@
-registry.svc.ci.openshift.org/openshift/tektoncd-v0.16.2:tektoncd-pipeline-controller quay.io/openshift-pipeline/tektoncd-pipeline-controller:v0.16.2
```
Comment (piyush-garg) [inline core-services/image-mirroring/tekton/mapping_tekton_v0_16_quay:1]: should not we add a new mapping?

**Review by piyush-garg — CHANGES_REQUESTED** [review]

File: core-services/image-mirroring/tekton/mapping_tekton_v0_16_quay (hunk 1 above)
Comment (nikhil-thomas) [reply core-services/image-mirroring/tekton/mapping_tekton_v0_16_quay:1]: it is there :)

**Review by piyush-garg — APPROVED** [review]
/lgtm

**Comment by chmouel** [comment]
/lgtm
/approve

Thanks 👍🏼🔥

**Pull request merged**